src/sequenceParser/Sequence.i

```diff
@@ -20,11 +20,15 @@ namespace std {
 
 %extend sequenceParser::Sequence
 {
-	%pythoncode
-	{
-		def __str__(self):
-			return self.string()
-	}
+        %pythoncode
+        {
+                def __str__(self):
+                    return self.string()
+                def __lt__(self, other):
+                    return self.getFilenameWithStandardPattern() < other.getFilenameWithStandardPattern()
+                def __eq__(self, other):
```

Comment:
it's a bit dangerous to have multiple comparison code (one  in c++ and one in python).